src/components/scan/hooks/useCameraCapture.ts

```
@@ -0,0 +1,217 @@
+		setCapturedImage(null);
+		setCapturedFile(null);
+
+		try {
+			const constraints = {
+				video: {
+					facingMode: { ideal: isMobile() ? 'environment' : 'user' },
+					...CAMERA_CONSTRAINTS,
+				},
+			};
+
+			const stream = await navigator.mediaDevices.getUserMedia(constraints);
```

ESLint

[error] Replace `·` with `⏎↹↹↹↹`

(prettier/prettier)